plugin/types.go

```diff
@@ -22,10 +22,11 @@ type Repo struct {
 // System provides important information about the Drone
 // server to the plugin.
 type System struct {
-	Version string   `json:"version"`
-	Link    string   `json:"link_url"`
-	Plugins []string `json:"plugins"`
-	Globals []string `json:"globals"`
+	Version         string   `json:"version"`
+	Link            string   `json:"link_url"`
+	Plugins         []string `json:"plugins"`
+	Globals         []string `json:"globals"`
+	EscalatePlugins []string `json:"escalate_plugins"`
```

Comment:
Yup. I'll duplicate it there. I made the change here because this is the package that `drone-exec` is using.